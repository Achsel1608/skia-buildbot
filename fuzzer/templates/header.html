<meta charset='utf-8' />
<meta name=viewport content='width=device-width, initial-scale=1'>

<link rel='stylesheet' href='/res/css/fuzzer.css' type='text/css'>

<script src="/res/js/core.js"></script>
<script src='/res/js/fuzzer.js' type='text/javascript'></script>
