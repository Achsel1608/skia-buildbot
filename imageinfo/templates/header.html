    <meta charset="utf-8">
    <meta name="theme-color" content="#526E9C">
    <link rel="shortcut icon" href="/res/img/favicon.ico" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <script src="/res/js/core.js" type="text/javascript" charset="utf-8"></script>
    <link href="/res/vul/elements.html" rel="import" />
    <style type="text/css" media="screen">
      body {
        margin: 0;
        padding: 0;
        transition: none;
        font-family: sans-serif;
      }

      header {
        background: #1f78b4;
        color: white;
        padding: 0.5em;
        margin: 0;
      }

      header > * {
        display: inline-block;
      }

      header h2 {
        margin: 0;
      }

      #main {
        padding: 1em;
      }

      #main pre,
      #main img{
        padding: 1em;
      }

      .visimage {
        width: 50%;
        min-width: 500px;
      }

      input {
        font-size: 15px;
        padding: 0.5em;
      }

    </style>
    <style type="text/css" is="custom-style" include="iron-flex iron-flex-alignment"></style>
