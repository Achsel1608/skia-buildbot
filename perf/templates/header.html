    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <script src="/res/js/perf.js"></script>
    <link rel="import" href="/res/imp/polymer/polymer.html">
    <link rel="import" href="/res/imp/paper-button/paper-button.html">
    <link rel="import" href="/res/imp/core-scaffold/core-scaffold.html">
    <link rel="import" href="/res/imp/core-header-panel/core-header-panel.html">
    <link rel="import" href="/res/imp/core-menu/core-menu.html">
    <link rel="import" href="/res/imp/core-item/core-item.html">
    <link rel="import" href="/res/imp/core-icon-button/core-icon-button.html">

    <link href="/res/imp/scaffold.html" rel="import" />
    <link href="/res/imp/query.html" rel="import" />
    <link href="/res/imp/login.html" rel="import" />
    <link href="/res/css/perf.css" rel="stylesheet" />


