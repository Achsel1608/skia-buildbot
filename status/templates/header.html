    <meta charset="utf-8">
    <meta name="theme-color" content="#526E9C">
    <link rel="shortcut icon" href="/res/img/favicon.ico" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <script src="/res/js/core.js"></script>
    <script type="text/javascript" src="https://www.google.com/jsapi"></script>
    <script type="text/javascript" src='https://www.google.com/jsapi?autoload={"modules":[{"name":"visualization","version":"1","packages":["corechart"]}]}'></script>

    <link href="/res/vul/elements.html" rel="import" />
