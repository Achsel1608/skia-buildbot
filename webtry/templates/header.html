    <meta charset='utf-8' />
    <meta name=viewport content='width=device-width, initial-scale=1'>

<link rel='stylesheet' href='/res/css/webtry.css' type='text/css'>
<link rel='stylesheet' href='/res/imp/bower_components/codemirror/lib/codemirror.css' type='text/css'>

<script src="/res/js/core.js"></script>
<script src='/res/js/webtry.js' type='text/javascript'></script>

<link href="/res/vul/elements.html" rel="import" />
