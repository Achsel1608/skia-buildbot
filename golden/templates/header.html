    <meta charset="utf-8">
    <meta name="theme-color" content="#FF7F00">
    <link rel="shortcut icon" href="/res/img/favicon.ico" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <script src="/res/js/core.js"></script>
    <link href="/res/css/core.css" rel="stylesheet" />
    <link href="/res/vul/elements.html" rel="import" />
